cuegui/cuegui/HostMonitor.py

```diff
@@ -256,6 +260,7 @@ def __refreshToggleCheckBoxSetup(self, layout):
 
     def __refreshToggleCheckBoxHandle(self, state):
         self.hostMonitorTree.enableRefresh = bool(state)
+        QtGui.qApp.settings.setValue("AutoRefreshMonitorHost", int(state))
```

Comment:
When I tested this change, when the checkbox is checked `state` here is equal to `2`, causing me to end up with a config file like this:

```
[General]
AutoRefreshMonitorHost=2
AutoRefreshMonitorProc=2
LastNotice=1587770087
<cut>
```

While the code works, I think it's best to ensure only `0` or `1` end up in that settings file as it's less confusing. A `2` would seem to imply that there's more than two options when there aren't.

Reply:
IMO it is because our checkbox has check-values 0 for 'False', 1 for 'Null' and 2 for 'True'. 
One solution is to pass bool() before int() i.e. int(bool(state))
Other is to change the check-value for states.
Using first solution will solve the problem even if I am wrong about the actual problem.

Reply:
@bcipriano sir,  changes done. Ready for review.